src/Network/AWS/S3/StreamingUpload.hs

```diff
@@ -201,23 +205,23 @@ multipart upload on any error, but this may also fail if, for example, the netwo
 connection has been broken. See `abortAllUploads` for a crude cleanup method.
```

Comment:
You can also set S3 buckets to clean up incomplete multipart uploads after a certain number of days, using a lifecycle policy.